src/Numerics/Integrate.cs

```diff
@@ -45,23 +46,11 @@ public static class Integrate
         /// <param name="intervalEnd">Where the interval stops, inclusive and finite.</param>
         /// <param name="targetAbsoluteError">The expected relative accuracy of the approximation.</param>
         /// <returns>Approximation of the finite integral in the given interval.</returns>
-        public static double OnClosedInterval(Func<double, double> f, double intervalBegin, double intervalEnd, double targetAbsoluteError)
+        public static double OnClosedInterval(Func<double, double> f, double intervalBegin, double intervalEnd, double targetAbsoluteError = 1E-8)
         {
             return DoubleExponentialTransformation.Integrate(f, intervalBegin, intervalEnd, targetAbsoluteError);
         }
 
-        /// <summary>
-        /// Approximation of the definite integral of an analytic smooth function on a closed interval.
-        /// </summary>
-        /// <param name="f">The analytic smooth function to integrate.</param>
-        /// <param name="intervalBegin">Where the interval starts, inclusive and finite.</param>
-        /// <param name="intervalEnd">Where the interval stops, inclusive and finite.</param>
-        /// <returns>Approximation of the finite integral in the given interval.</returns>
-        public static double OnClosedInterval(Func<double, double> f, double intervalBegin, double intervalEnd)
-        {
-            return DoubleExponentialTransformation.Integrate(f, intervalBegin, intervalEnd, 1e-8);
-        }
```

Comment:
It seems removing this overload would be a breaking change (optional arguments are not actually optional once compiled). I think this should be reverted.